Sources/SwifterSwift/SwiftStdlib/StringExtensions.swift

```diff
@@ -957,6 +957,17 @@ public extension String {
         return range(of: pattern, options: .regularExpression, range: nil, locale: nil) != nil
     }
     #endif
+    
+    #if canImport(Foundation)
```

Comment:
String changes are still here...

Reply:
As @guykogus suggested I separated these 2 extensions and made 2 separate PRs. I think this PR could be closed as those other two exist.